### PR TITLE
Hardik branch

### PR DESCRIPTION
**Pull request by hrsautomation20**
**Created**
Router.get under display.js for bookresults
bookresult.handlebars

**Updated/Modified**
searchbooks.handlebars 
searchbooks.js
style.css - updated to hide search result tag

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -23,7 +23,7 @@ body {
 .container {
     margin-top: 10%;
     margin-bottom: 10%;
-
```
Comment (hrsautomation20) [inline public/css/style.css:26]: Auto fixed by prettier formatter

File: public/css/style.css [hunk 2]
```diff
@@ -67,7 +67,7 @@ body {
     backdrop-filter: blur(5px);
 }
 
-.image-overlay > * {
```
Comment (hrsautomation20) [inline public/css/style.css:70]: Auto fix formatting by prettier

File: public/css/style.css [hunk 3]
```diff
@@ -76,11 +76,17 @@ body {
     opacity: 1;
 }
 
-.image-overlay:hover > * {
```
Comment (hrsautomation20) [inline public/css/style.css:79]: Auto formatted by prettier

File: public/css/style.css [hunk 4]
```diff
@@ -76,11 +76,17 @@ body {
...
 
 .book-description {
     font-size: 1.25em;
     margin-top: 0.25em;
+}
+
+/* BOOK RESULT  */
+.book-list {
+    margin-top: 50px;
+    visibility: hidden;
```
Comment (hrsautomation20) [inline public/css/style.css:91]: Added .book-list to hide the visibility

File: public/js/searchbooks.js [hunk 5]
```diff
@@ -1,5 +1,12 @@
 $(document).ready(function () {
-  var item, tile, author, publisher, bookLink, bookImg;
+  var item,
+    title,
+    author,
+    publisher,
+    bookLink,
+    bookImg,
+    publishedDate,
+    pageCount;
```
Comment (hrsautomation20) [inline public/js/searchbooks.js:9]: Spelling correct, formatted and added PublishedDate and PageCount

File: views/bookresult.handlebars [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+      style="width: 1000px; height: 700px"
+    ></div>
+  </div>
+  <link
+    rel="stylesheet"
+    href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+    integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO"
+    crossorigin="anonymous"
+  />
+</body>
```
Comment (hrsautomation20) [inline views/bookresult.handlebars:28]: Added new file

**Review by smithnaomi — APPROVED** [review]

**Pull request merged**